src/index.js

```diff
@@ -5,82 +5,92 @@ const lineReader = require('line-reader'); // to read line one by one
 const fetch = require("node-fetch"); // to send request and get response
 // to match url with http and https
 const regex = new RegExp(/https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/g);
-//set the arguement
-const argv  = require("yargs") 
-    .scriptName("fbl")
-    .usage("Usage: $0 [options] <argument> where argument can be a file name or a url")
-    .example(
-        "$ fbl -f test.html or test.txt",
-        "process the file to find any broken link inside the file."
-    )
-    .example(
-        "$ fbl -u https://www.google.ca/",
-        "check the url if it is broken link or not."
-    )
-    .option("f", {
-        alias: "fileName",
-        describe: "Any file you want to check for broken link",
-        type: "array",
 
-    })
-    .option("u", {
-        alias: "url",
-        describe: "Any url you want to check for broken link",
-        type: "string",
+const opts = process.argv.slice(2);
 
-    })
-    .check((argv) => {
-        if(argv.f || argv.u){
-            handleArg(argv)
-            return true
-       }
-        console.log(chalk.red.bold('At least one arguement is required!'));
-        return false
-    })
-    .version('v', 'Show version number', '1.0.0')
-    .alias('v', 'version')
-    .help('h', "Show help.")
-    .alias('h', 'help')
-    .epilog("copyright 2020")
-    .argv;
+if (opts[0] == "/v" || opts[0] == "/version") {
+    console.log("Current version: 1.0.0");
+} else {
```

Comment:
`Yargs` is been used for managing command options. We should be using this for all options instead of using `process.argv`.